bee-hive/bee_hive/run_workflow.py

```diff
@@ -25,19 +31,129 @@ def load_agent(agent):
         agent_store = json.load(f)
     return agent_store[agent]
 
-
+# --- Abstract Base Class for Agent Runners ---
+class AgentRunner(ABC):
+    def __init__(self, agent_name):
+        self.agent_name = agent_name
+
+    @abstractmethod
+    def run(self, prompt):
```

Comment:
Can we enforce a return value for the `run(..) -> dict`?

Reply:
yes, still need to do this.

Reply:
Added
- #131 : simple workflow for ad-hoc testing
- #127 : automated testing (local/pr/ci)